blocks/heading/heading.css

```diff
@@ -18,6 +18,11 @@
     line-height: 31.36px;
 }
 
+html[dir="rtl"] .heading-wrapper .heading h2{
+text-align: right;
```

Comment:
I think if you use `text-align: end`, it'll work for both rtl and non-rtl without any extra styles. Can you try?

Reply:
sure